charts/base-cluster/README.md.gotmpl

```diff
@@ -19,21 +19,23 @@ git init
 flux create helmrelease --export base-cluster -n flux-system --source HelmRepository/teuto-net.flux-system --chart base-cluster --chart-version 3.x.x > cluster.yaml
 
 # maybe use the following name for your cluster;
-kubectl get node -o json | jq '.items[0].metadata.annotations["cluster.x-k8s.io/cluster-name"]'
+kubectl get node -o jsonpath='{.items[0].metadata.annotations.cluster\.x-k8s\.io/cluster-name}'
```

Comment:
Oh this is cool. No need for jq anymore?

Reply:
Yessireeeeee!